plugin/lsp.vim

```diff
@@ -172,3 +173,11 @@ nnoremap <plug>(lsp-status) :<c-u>echo lsp#get_server_status()<cr>
 nnoremap <plug>(lsp-next-reference) :<c-u>call lsp#internal#document_highlight#jump(+1)<cr>
 nnoremap <plug>(lsp-previous-reference) :<c-u>call lsp#internal#document_highlight#jump(-1)<cr>
 nnoremap <plug>(lsp-signature-help) :<c-u>call lsp#ui#vim#signature_help#get_signature_help_under_cursor()<cr>
+
+nnoremap <expr> <plug>(lsp-floatwin-scroll-up) lsp#internal#ui#floatwin#scroll(-g:lsp_floatwin_scroll_delta)
+nnoremap <expr> <plug>(lsp-floatwin-scroll-down) lsp#internal#ui#floatwin#scroll(g:lsp_floatwin_scroll_delta)
+inoremap <expr> <plug>(lsp-floatwin-scroll-up) lsp#internal#ui#floatwin#scroll(-g:lsp_floatwin_scroll_delta)
+inoremap <expr> <plug>(lsp-floatwin-scroll-down) lsp#internal#ui#floatwin#scroll(g:lsp_floatwin_scroll_delta)
+xnoremap <expr> <plug>(lsp-floatwin-scroll-up) lsp#internal#ui#floatwin#scroll(-g:lsp_floatwin_scroll_delta)
+xnoremap <expr> <plug>(lsp-floatwin-scroll-down) lsp#internal#ui#floatwin#scroll(g:lsp_floatwin_scroll_delta)
```

Comment:
Sorry... I don't know `up/down` is better name or not.

Reply:
I'm wondering if we should only expose a function instead.

```
lsp#can_scroll()
lsp#scroll(-1)
lsp#scroll(1)
```

in the future we could expose method to only scroll a particular window. `lsp#scroll(-1, id)`.

The reason I was preferring function is so that the user can pass custom values on how much they want to scroll whether it is 5 rows or 1 row at a time.

Thoughts?

Reply:
That makes sense.
In particular, the `id` would be needed maybe.

OK, I will remove `g:lsp_floatwin_scroll_delta` and introduce `lsp#scroll` as public api.

Reply:
`id` support we can add latter. for now most people will only have 1 popup open and in worst case both popup will scroll. We can solve this issue later if more people file bugs.